SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -121,14 +121,16 @@ - (instancetype)initDefault
         // The best we can try to do is to discard the current database
         // and behave like a clean install.
 
-        OWSProdError(@"storage_error_could_not_load_database");
+        // NOTE: This analytics event will never be delivered, since the database isn't working.
+        OWSProdCritical(@"storage_error_could_not_load_database");
```

Comment:
Yeah, I've been thinking about it, but it would probably require a whole new set of database-specific macros.  I'm wondering if we'll use the "critical-severity" macros anywhere else.